app/models/test_track/split_registry.rb

```diff
@@ -27,9 +27,9 @@ def weights_for(split_name)
     registry_hash && registry_hash['splits'][split_name] && registry_hash['splits'][split_name]['weights'].freeze
   end
 
-  def to_v1_hash
+  def to_hash
     registry_hash && registry_hash['splits'].each_with_object({}) do |(k, v), result|
-      result[k] = v['weights']
+      result[k] = { weights: v['weights'], feature_gate: v['feature_gate'] }
```

Comment:
so the new JS client code consumes `splits` from `window.TT` in this new format?

Reply:
Exactly. This is one of the breaking changes included in the 2.0.0 major update.